crates/blockchain-tree/src/config.rs

```diff
@@ -35,11 +39,17 @@ impl BlockchainTreeConfig {
         max_reorg_depth: u64,
         max_blocks_in_chain: u64,
         num_of_additional_canonical_block_hashes: u64,
+        max_unconnected_blocks: u64,
```

Comment:
let's make this usize since we cast downstream anyway

Reply:
Was on the fence on this, as i wanted to have consistent types in config so i put u64. `usize` it is.